package.json

```diff
@@ -1,46 +1,17 @@
 {
   "name": "plugin-agent-autoresponse",
-  "version": "0.0.0",
+  "version": "0.0.2",
```

Comment:
hmm, not sure why it changed the version

Reply:
You probably did a deploy?

Reply:
oh maybe i did when we were testing a while back. i'll change it back

Reply:
Yes, it's time to do it. It does not allow Deploy and when updating the version it fails.

Reply:
this will be merged tomorrow!